src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[]
+  selectTodo: (todoIDfromlist: number | string) => void
+  selectedTodoID:number | string
```

Comment:
```suggestion
  selectTodo: (todoIdFromList: number | string) => void
  selectedTodoId:number | string
```
Use camelCase here as well